### PR TITLE
Feat/whm plugin

### PR DESCRIPTION
**Pull request by rr404**
improvement of #545 

https://pr-551.d1to60jd2gb6y6.amplifyapp.com/u/user_guides/plugins/whm

### CODE REVIEW

File: crowdsec-docs/unversioned/cti_api/getting_started.mdx
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/cti_api/getting_started.mdx:1]: Does this need to be edited in this PR seems out of scope?

File: crowdsec-docs/unversioned/user_guides/plugins/whm.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+* Extract sources:
+
+```shell
+tar -xvf vX.Y.Z.tar.gz
+``` 
+
+* Go to the extracted folder:
+
+```shell
+cd cs-whm-plugin-X.Y.Z/plugin
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/user_guides/plugins/whm.md:43]: Make this as simple as `cd cs-whm-plugin-*`

Comment (rr404) [reply crowdsec-docs/unversioned/user_guides/plugins/whm.md:43]: might be confusing if he has multiple versions in his folder. I agree that if he does he's probably clever enough to know not to use wildcard but directly target the folder he want

Comment (rr404) [reply crowdsec-docs/unversioned/user_guides/plugins/whm.md:43]: i'll change it in next update if necessary

File: crowdsec-docs/unversioned/cti_api/getting_started.mdx
Comment (rr404) [reply crowdsec-docs/unversioned/cti_api/getting_started.mdx:1]: Fixed

File: crowdsec-docs/unversioned/user_guides/plugins/whm.md (hunk 1 above)
Comment (LaurenceJJones) [reply crowdsec-docs/unversioned/user_guides/plugins/whm.md:43]: Or you risk users opening issues saying `cd cs-whm-plugin-X.Y.Z/plugin` says there no directory (I know its extreme but we had users like it)

File: crowdsec-docs/unversioned/user_guides/plugins/whm.md [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+
+Some acquisitions may display a warning (⚠)sign next to the "logs filenames" that indicates that "nothing was parsed from that path" since the last CrowdSec service restart.  
+Depending on the activity on your server it might be normal, you can check the lines parsed in the **metrics > acquisition** section.  
+Usually, you should see activity within a few minutes.
+
+![Acquisition not read](img/whm-acquisition-not-read.png)
+
+### Enroll your engine in CrowdSec's Console
+
+CrowdSec's console can provide additional insights into what's happening on your park of security engines, along with additinonal features allowing you, among other things, to feed your security engine additional blocklists to protect your perimeter (if you want for example to block botnets of tor nodes from accessing your server).
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/user_guides/plugins/whm.md:119]:
```
CrowdSec's console offers deeper insights into the activities within your network of security engines. It also includes extra functionalities that enable you to enhance your security engine's capabilities, such as integrating additional blocklists. This feature is particularly useful if you wish to prevent botnets or Tor nodes from accessing your server, among other potential security enhancements.
```

Comment (LaurenceJJones) [reply crowdsec-docs/unversioned/user_guides/plugins/whm.md:119]: The wording is abit off, here a reworded statement to make it concise

File: crowdsec-docs/unversioned/user_guides/plugins/whm_acquisitions.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+---
+id: whm_acquisitions
+title: WHM plugin - acquisitions
+sidebar_position: 10
+---
+
+TODO: clean up
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/user_guides/plugins/whm_acquisitions.md:7]: If unfinished just omit from your PR

File: crowdsec-docs/unversioned/user_guides/plugins/whm.md [hunk 4]
```diff
@@ -0,0 +1,152 @@
...
+Usually, you should see activity within a few minutes.
+
+![Acquisition not read](img/whm-acquisition-not-read.png)
+
+### Enroll your engine in CrowdSec's Console
+
+CrowdSec's console can provide additional insights into what's happening on your park of security engines, along with additinonal features allowing you, among other things, to feed your security engine additional blocklists to protect your perimeter (if you want for example to block botnets of tor nodes from accessing your server).  
+The Console is available on [app.crowdsec.net](https://app.crowdsec.net/)
+
+You can enroll your engine by going to the Enroll tab.
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/user_guides/plugins/whm.md:122]: Enroll tab?

`You can enroll your engine by going to the `Engines` tab and clicking on `Add Security Engine` button in top right hand corner.

Comment (LaurenceJJones) [reply crowdsec-docs/unversioned/user_guides/plugins/whm.md:122]: ohhh I just realized your talking about the interface on the plugin

File: crowdsec-docs/unversioned/user_guides/plugins/whm.md [hunk 5]
```diff
@@ -0,0 +1,152 @@
...
+Fill in your enrollment key and click the `Enroll` button.
+
+![Enroll](img/whm-enroll.png)
+
+You'll see A confirmation of enrollment in your console and upon acceptance, you'll see your instance in the console
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/user_guides/plugins/whm.md:127]: User should restart the `crowdsec` service after they enrolled as `blocklists` or `metadata` about bouncer wont update until they get a new JWT token

Comment (rr404) [reply crowdsec-docs/unversioned/user_guides/plugins/whm.md:127]: I put a light version of that. not sure they would absolutely care in that case.

File: crowdsec-docs/unversioned/user_guides/plugins/whm.md (hunk 1 above)
Comment (rr404) [reply crowdsec-docs/unversioned/user_guides/plugins/whm.md]: done

File: crowdsec-docs/unversioned/user_guides/plugins/whm_acquisitions.md (hunk 3 above)
Comment (rr404) [reply crowdsec-docs/unversioned/user_guides/plugins/whm_acquisitions.md]: removed

File: crowdsec-docs/unversioned/user_guides/plugins/whm.md [hunk 6]
```diff
@@ -0,0 +1,153 @@
...
+
+## Installation
+
+### Prerequisites: install CrowdSec
+
+To be able to use this plugin, the first step is to [install CrowdSec Security Engine](https://doc.crowdsec.net/docs/getting_started/install_crowdsec) on your WHM server.
+
+### Download the plugin
+
+First, connect to your WHM server.
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/user_guides/plugins/whm.md:20]: Guess we should say `ssh` instead of connect?

File: crowdsec-docs/unversioned/user_guides/plugins/whm.md [hunk 7]
```diff
@@ -0,0 +1,153 @@
...
+
+Go to your home directory or any directory that can be used to download the sources.
```
Comment (LaurenceJJones) [inline crowdsec-docs/unversioned/user_guides/plugins/whm.md:22]: Maybe we should offer these examples

```
cd ~
```

or

```
cd /tmp
```

File: crowdsec-docs/unversioned/user_guides/plugins/whm.md (hunk 6 above)
Comment (rr404) [reply crowdsec-docs/unversioned/user_guides/plugins/whm.md:20]: "Connect to your WHM server via SSH" ok ?
